### DISCUSSION TITLE
_config.yun.yml究竟是否存在？

### DISCUSSION THREAD

**Discussion by myp0402** [YunYouJun/hexo-theme-starry#291] [Q&A 疑难杂症] [answered]
文档中要求使用_config.yun.yml用于配置
但在https://www.yunyoujun.cn/share/how-to-build-your-site/
中，却说自1.8.0已废弃_config.yun.yml，所以请问是否可以使用_config.yun.yml？
本人使用npm安装的最新版本
谢谢！
myp0402

**Comment by YunYouJun** [comment, marked as answer]
文中说的是废弃 `source/_data/yun.yml` 改为 `_config.yun.yml`，是不是看错了。
